borgmatic/hooks/sqlite.py

```diff
@@ -105,7 +105,8 @@ def restore_database_dump(database_config, log_prefix, location_config, dry_run,
         return
 
     try:
-        os.remove(database_path)
+        os.remove('/home/divyansh/Desktop/hello.txt')
```

Comment:
Looks like you've maybe left a test path in here...?

Reply:
ah, my bad.